.changelog/82ce917a265846b68d6977bd88247343.json

```diff
@@ -0,0 +1,9 @@
+{
+    "id": "82ce917a-2658-46b6-8d69-77bd88247343",
+    "type": "feature",
```

Comment:
nit and not a blocker, but it seems weird to have "feature" for a deprecation :P